tox.ini

```diff
@@ -155,7 +155,11 @@ basepython = python3
 skip_install = True
 deps =
     mypy
-    git+https://github.com/Shoobx/mypy-zope
+    mypy-zope
+    types-mock
```

Comment:
For explanation of "types-*" packages see http://mypy-lang.blogspot.com/2021/06/mypy-0900-released.html